src/ethz_snow/config/snowConfig_default.yaml

```diff
@@ -1,36 +1,87 @@
+# temperature resolution within vial (homogeneous, spatial_1D, spatial_2D)
+temperature: spatial_2D
```

Comment:
I feel that this is not a informative name for this key. How about `temp_model`?

Reply:
or `temp_dim` or `dimensionality`

Reply:
Fully agree, changed to dimensionality.